Automattic/jetpack
# MU WPCOM: Add description link to Subscriber Login block

Fixes https://github.com/Automattic/jetpack/issues/38592

## Proposed changes:

It adds a description link to the Subscriber Login block.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Edit a post
* Insert the Subscriber Login block
* Click it and make sure the link shows up
